go.mod

```diff
@@ -64,6 +53,7 @@ require (
 	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
 	golang.org/x/sys v0.0.0-20190613124609-5ed2794edfdc // indirect
 	gomodules.xyz/cert v1.0.0
+	gomodules.xyz/nats-logr v0.0.0-20190702081800-7457beea95bc
```

Comment:
this should be changed to master/release after `gomodules.xyz/nats-logr` is released